homeassistant/components/doods/image_processing.py

```diff
@@ -0,0 +1,366 @@
+"""Support for the DOODS service."""
+import logging
+import time
+import voluptuous as vol
```

Comment:
Please add a blank line between standard library and 3rd party import groups.

Reply:
Okay, that's done.

Reply:
voluptuous is 3rd party.

Reply:
Ahh right. Updated that too.